package.json

```diff
@@ -22,6 +22,7 @@
     "test": "npm run tests && npm run lint"
   },
   "dependencies": {
+    "lodash": "4.17.4",
```

Comment:
probably want this to be more general, so it can reuse the lodash already bundled by wp-calypso. maybe "^4.0"?